modules/ROOT/pages/security/secure-connections.adoc

```diff
@@ -53,174 +59,90 @@ A single private link connection applies to all instances in a region.
-
-Once activated, you can create an endpoint in your VPC that connects to Aura.
-
-For a step-by-step guide, see the link:https://neo4j.com/blog/neo4j-aws-privatelink-configuration/[How to Configure Neo4j Aura With AWS PrivateLink] blog article.
-
-image::privatelink.png["VPC connectivity with AWS PrivateLink", title="VPC connectivity with AWS PrivateLink"]
-
-All applications running Neo4j workloads inside the VPC are routed directly to your isolated environment in Aura without traversing the public internet.
-You can then disable public traffic, ensuring all traffic to the instance remains private to your VPC.
-
 [NOTE]
 ====
-* PrivateLink applies to all instances in the region.
-* When activated, a *Private Connection* label, shield icon, and dedicated *Private URI* will appear on any instance tile using PrivateLink in the Aura Console.
-* If you disable public traffic, you must use a dedicated VPN to connect to your instance via Browser or Bloom.
-* Connections using private endpoints are one-way.
-Aura VPCs can't initiate connections back to your VPCs.
-* In AWS region us-east-1, we do not support the Availability Zone with ID use1-az3 for private endpoints.
+* Private endpoints apply to all instances in the region.
+* When activated, a Private Connection label, shield icon, and dedicated Private URI will appear on any instance tile using a private endpoint in the Aura console.
+* Connections using private endpoints are one-way. Aura VPCs can’t initiate connections back to your VPCs.
 ====
 
-==== Browser and Bloom access over private endpoints
-
-To connect to your instance via Browser or Bloom, you must use a dedicated VPN.
-This is because when you disable public access to your instance, this applies to all connections, including those from your computer when using Browser or Bloom.
-
-Without private endpoints, you access Browser and Bloom over the internet:
-
-image::privatelink_01_before_enabling.png["Architecture overview before enabling private endpoints", title="Architecture overview before enabling private endpoints"]
-
-When you have enabled private endpoints **and** disabled public internet access, you can no longer connect Browser or Bloom to your instances over the internet:
-
-image::privatelink_02_enabled_private_traffic_only.png["Architecture overview with private endpoints enabled and public traffic disabled", title="Architecture overview with private endpoints enabled and public traffic disabled"]
+=== AWS private endpoints
 
-To continue accessing Browser and Bloom, you can configure a VPN (Virtual Private Network) in your VPC and connect to Browser and Bloom over the VPN.
+For a step-by-step guide, see the link:https://neo4j.com/blog/auradb/neo4j-aws-privatelink-configuration/#2[How to Configure Neo4j Aura With AWS PrivateLink] blog article.
+Refer to link:https://aws.amazon.com/privatelink[AWS PrivateLink] docs for IAM requirements.
 
```

Comment:
This piece of information seems to have gone missing:
_All applications running Neo4j workloads inside the VPC are routed directly to your isolated environment in Aura without traversing the public internet. You can then disable public traffic, ensuring all traffic to the instance remains private to your VPC._